src/fluree/db/serde/json.cljc

```diff
@@ -11,30 +11,29 @@
   [block]
   (assoc block :flakes (mapv flake/parts->Flake (:flakes block))))
 
-
-
 (defn- deserialize-child-node
   "Turns :first and :rhs into flakes"
   [child-node]
-  (assoc child-node :first (some-> (:first child-node)
-                                   (flake/parts->Flake))
-                    :rhs (some-> (:rhs child-node)
-                                 (flake/parts->Flake))))
+  (assoc child-node
+         :first (some-> child-node :floor flake/parts->Flake)
```

Comment:
Is this supposed to be `:floor`? Just checking because I thought it was `:first`.

Reply:
you're right! I changed the name to floor to not clobber the `clojure.core/first` function, but later changed my mind because we never actually used the `first` function in any code where we destructured the `first` flake. I must have missed this when I changed it back. Thanks for the extra 👀